### DISCUSSION TITLE
Block entering skin editor via shortcut during gameplay (in line with other overlays)

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#24509] [Q&A]
https://github.com/ppy/osu/assets/53081750/7418e3d2-9af7-4ec2-ba2a-70f5aad49e21

**Comment by Theighlin** [comment]
Bumping this because i don't know if the situation here got worse now that skin editor goes back when it reaches the end, and it should be easy enough thing to fix.

This overlay should probably be blocked on pause and break times too because i can't see a reason why someone would open it in that moment.

**Reply by verarr** [reply]
> This overlay should probably be blocked on pause and break times too because i can't see a reason why someone would open it in that moment.

It's easier to quickly iterate over small adjustments if something feels off while actually trying the skin.

**Reply by Theighlin** [reply]
~I guess you can only do this with relax? You can't feel the skin because entering the editor will make you fail, but maybe it's faster to return to the gameplay with the shortcut and press retry?~

Ooops, No Fail is a thing.

Though this behaviour still feels kinda weird to me, would like to hear more opinions on this.

**Reply by verarr** [reply]
I mainly meant making adjustments while paused, but those mods are an option too.

Also

> i don't know if the situation here got worse now that skin editor goes back when it reaches the end

AFAIK it only does that if you enter it from the main menu (or the "_Gameplay_" button too? idk about that one.) It won't do that if you actually started the play yourself.
